NeuroID/src/advancedDeviceLib/java/com/neuroid/tracker/extensions/AdvancedDeviceExtension.kt

```diff
@@ -59,6 +60,7 @@ fun NeuroIDPublic.startSession(
 }
 
 fun NeuroID.captureAdvancedDevice(shouldCapture: Boolean) {
+    captureEvent(type = LOG, m = "isAdvancedDevice setting: $shouldCapture", level = "INFO")
```

Comment:
can we also add a log statement in the configure when they set the global isAdvancedDevice?

Reply:
added

Reply:
sorry this is NIT but can this log read `shouldCapture` because they can pass in an overriding parameter and not just the `isAdvancedDevice` parameter from the global state and if we have both I'd like to know which is which

Reply:
good idea. will do

Reply:
updated